pkg/scheduler/api/job_info.go

```diff
@@ -107,6 +107,7 @@ type TaskInfo struct {
 
 	Name      string
 	Namespace string
+	TaskSpec  string // value of "volcano.sh/task-spec"
```

Comment:
`taskSpec` is the task name. Is it the same with `Name` field.

Reply:
Yes, have updated to TaskRole.